tests/console/yast2_vnc.pm

```diff
@@ -27,17 +28,18 @@ sub configure_remote_admin {
     send_key $remote_admin{allow_remote_admin_with_session};
     assert_screen 'yast2_vnc_allow_remote_admin_with_session';
     # Firewall Settings
-    if (check_screen 'yast2_vnc_firewall_port_closed') {
+    assert_screen([qw(yast2_vnc_no_network_interface yast2_vnc_firewall_port_closed yast2_vnc_firewall_port_open)]);
+    if (is_tumbleweed && (match_has_tag('yast2_vnc_no_network_interface'))) {
+        send_key $cmd{next};
+    }
+    if (match_has_tag('yast2_vnc_firewall_port_closed')) {
         send_key $firewall_settings{open_port};
+        check_firewall_port_details();
     }
-    # Firewall Details
-    assert_screen 'yast2_vnc_firewall_port_open';
-    send_key $firewall_settings{details};
-    assert_screen 'yast2_vnc_firewall_port_details';
-    send_key $firewall_details{network_interfaces};
-    assert_screen 'yast2_vnc_firewall_details_interface_selected';
-    send_key $cmd{ok};
-    assert_screen 'yast2_vnc_firewall_details_selected';
+    if (match_has_tag('yast2_vnc_firewall_port_open')) {
```

Comment:
cannot be open and closed at the same time, guess this should be `ifelse`, but then the firewall details should be out.

Reply:
> cannot be open and closed at the same time, guess this should be `ifelse`, but then the firewall details should be out to not repeat it?

The original logic is that if it is closed, then open it, and then check the firewall details;
If it is opened by default, then check the firewall details directly.
So need to use ifelse or not?

Reply:
you don't need to run `match_has_tag` again, so I guess and elseif would be more efficient and check info can be done outside if/elseif as it shared by both.